server/graphql/dog-licenses.js

```diff
@@ -8,40 +7,63 @@ import type {
 
 export const Schema = `
```

Comment:
Since you’re updating the GraphQL bits, run:

`yarn gulp graphql:schema`

to update the auto-generated schema descriptions.

In future, you can have `yarn gulp watch` going in a terminal window to update them automatically on code changes.